src/tree/tree.scss

```diff
@@ -0,0 +1,149 @@
+
+// Spacing for when expander button is not present: (icon margin 6px) + (icon width 16px) + (button padding 4px) + (button border 2px)
+$iui-expander-button-width: 28px;
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  list-style: none;
+  @include iui-reset;
+}
+
+@mixin iui-sub-tree {
+  list-style: none;
+  @include iui-reset;
+}
+
+@mixin iui-tree-node {
+  display: flex;
+  cursor: pointer;
+  padding: 0 $iui-s;
+
+  &-checkbox {
+    margin-right: $iui-s;
+  }
+
+  &-content {
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    min-height: $iui-baseline * 3;
+    margin-left: calc(28px * (var(--level, 0)));
+    overflow: hidden;
+    padding-left: $iui-xxs;
+
+    &-icon {
+      @include iui-icons-default;
+      padding: 0 6px;
+      flex-shrink: 0;
+
+      &:first-child {
+        margin-left: $iui-expander-button-width;
+      }
+    }
+
+    &-expander {
+      > .iui-button-icon {
+        @media (prefers-reduced-motion: no-preference) {
+          transition: transform $iui-speed-fast ease-out;
+        }
+      }
+    }
+
+    &-label {
+      min-width: 0;
+      padding-left: 6px;
+
+      &:first-child {
+        margin-left: $iui-expander-button-width;
+      }
+    }
+
+    &-title,
+    &-caption {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    &-title {
+      font-size: $iui-font-size;
+    }
+
+    &-caption {
+      font-size: $iui-font-size-small;
+      @include themed {
+        color: t(iui-text-color-muted);
+      }
+    }
+  }
+
+  &:focus {
+    @include themed {
+      outline: thin solid rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
+      outline-offset: -1px;
+    }
+  }
+
+  &.iui-active {
+    &,
+    &:hover {
+      @include themed {
+        background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+        outline: $iui-active-outline;
+        outline-offset: -1px;
+      }
+    }
+
+    &:focus {
+      outline-width: $iui-xxs;
+      outline-offset: -2px;
+    }
+  }
+
+  &.iui-disabled {
+    &,
+    &:hover {
+      cursor: not-allowed;
+      outline: none;
+      @include themed {
+        background-color: t(iui-color-background-1);
+        color: t(iui-text-color-placeholder);
+      }
+
+      .iui-tree-node-content-caption {
+        @include themed {
+          color: t(iui-text-color-placeholder);
+        }
+      }
+    }
+  }
+
+  &:hover {
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+    }
+
+    .iui-tree-node-content-caption {
+      @include themed {
+        color: t(iui-color-foreground-body);
+      }
+      @media (prefers-reduced-motion: no-preference) {
+        transition: color $iui-speed-fast ease;
+      }
+    }
+  }
+
+  &.iui-expanded {
+    > .iui-tree-node-content > .iui-tree-node-content-expander > .iui-button-icon {
```

Comment:
This long selector can be fixed by giving a modifier to `iui-button-icon`

Reply:
changed to `iui-expander-icon`

Reply:
That's not really helping, because now that modifier is applied on all expanders. What you need is a class that will only be used to rotate the icon.

Reply:
What do you mean? Should it be something like iui-tree-node-content-expander-icon?

Reply:
what I mean is that selector still has 0-4-0 specificity (previously it had 0-5-0 but this decrement is not because off the new classname).

Yes `iui-tree-node-content-expander-icon` might help, because then you won't need to nest it inside `iui-expanded`

Reply:
Changed it to use only `iui-tree-node-content-expander-icon`

Reply:
Maybe I'm not being clear. I'm trying to say you don't need `iui-expanded` at all. It's adding unnecessary nesting and not doing anything else. So you could just have a class whose sole purpose is to rotate the icon. That way its specificity will remain low.

Reply:
If I remove iui-expanded how/where would I do the rotation?

Reply:
You can do the rotation by toggling a class on the icon itself.

Example:
![image](https://user-images.githubusercontent.com/9084735/144885548-a03b5c74-358d-457e-9c4f-08fed03cefe6.png)

This does make the class name super long but the specificity stays at 0-1-0. You can also do `&.iui-expanded` instead of `&-expanded` if you don't like the long class; it does increase the specificity to 0-2-0 but it's better than current situation.

Also unrelated but I noticed an extra (unused) class in your HTML which you may want to remove
![image](https://user-images.githubusercontent.com/9084735/144886160-76e3e6ed-e7f6-4c9b-b273-ba97e230c515.png)

Reply:
Removed `iui-expanded` and `iui-tree-node-content-expander`